vividus/src/main/java/org/vividus/runner/GenericRunner.java

```diff
@@ -51,6 +52,8 @@
     private final Embedder embedder;
     private final IBatchedPathFinder batchedPathFinder;
 
+    private Map<String, List<String>> batchToStoryPaths;
```

Comment:
this logic should be moved to `BatchedPathFinder`

Reply:
Why?  It still won't be clear from the main logic that this value gets cached, so each time someone gets to this point he will have to lookup BatchedPathFinder class